Some unit-tests expect an installed version of Gambit, redirect to the
local source.

--- a/tests/run-unit-tests.scm
+++ b/tests/run-unit-tests.scm
@@ -184,7 +184,7 @@
 
       (case target
         ((C)
-         (run "../gsi/gsi" "-:debug-settings=-,io-settings=lu,~~=.." "-f" file))
+         (run "../gsi/gsi" "-:debug-settings=-,io-settings=lu,~~=..,~~lib=../lib" "-f" file))
         (else
          (let ((gsi (string-append "../gsi/gsi-" (symbol->string target))))
            (run gsi "-f" file))))))
